stubs/requests/requests/sessions.pyi

```diff
@@ -40,10 +41,23 @@ def merge_setting(request_setting, session_setting, dict_class=...): ...
 def merge_hooks(request_hooks, session_hooks, dict_class=...): ...
 
 class SessionRedirectMixin:
-    def resolve_redirects(self, resp, req, stream=..., timeout=..., verify=..., cert=..., proxies=...): ...
+    def resolve_redirects(
+        self,
+        resp,
+        req,
+        stream: bool = ...,
+        timeout=...,
+        verify=...,
+        cert=...,
+        proxies=...,
```

Comment:
```suggestion
        timeout: Incomplete | None = ...,
        verify: Incomplete | None = ...,
        cert: Incomplete | None = ...,
        proxies: Incomplete | None = ...,
```

Reply:
I'd prefer we just type these rather than resorting to incomplete unnecessarily

Reply:
Of course we should aim to have these functions completely typed eventually. But we don't need to do that in _this_ PR; and this is easy to verify, and better than the status quo